src/Extensions/Abstractions/EventSourcesEventIds.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.Omex.Extensions.Abstractions
+{
+	/// <summary>
+	/// Event ids enum
+	/// </summary>
```

Comment:
If this can't be reordered, please add a `<remarks>` tag with the information. The ordering at the moment seems somewhat random and begging for someone to reorder it. So if they're not supposed to, we should explain why.

Reply:
I will resolve it in the follow-up pr https://github.com/microsoft/Omex/pull/168